### PR TITLE
meaningful /quickquotecf URL name that stands for the Credit File

### PR DESCRIPTION
**Pull request by KastorDriver**
#### Description
[GRO-1815](https://selina.atlassian.net/browse/GRO-1815)

#### Background Context
We have the next endpoints
* `/application/dipcc` - this one fetches a credit file and checks credit commitments inside it
* `/application/dip` - this one fetches a credit file as well, but doesn't do anything with credit commitments inside the file
* `/application/quickquote `- this one doesn't fetch a credit commitment file
* `/application/quickquotecc` - this one fetches a credit commitment file, but doesn't care about credit commitments!

So, to make the naming less confusion we need to change the `/application/quickquotecc` to the `/application/quickquotecf` which means Credit File. So, it's clear that the last endpoint fetches a credit file, but doesn't operate with credit commitments.

Also, I changed the permission's name to trigger the endpoint from the QQ_CC to the QQ_CF.

#### Additional Information

[GRO-1815]: https://selina.atlassian.net/browse/GRO-1815?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **kastordriver**
You can assign the PR to them by writing `/assign @kastordriver` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/com/selina/lending/internal/service/permissions/annotation/Permission.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
     Scope scope();
 
     enum Resource {
-        DIP, DIP_CC, CC, QQ, QQ_CC, APPLICATION
+        DIP, DIP_CC, CC, QQ, QQ_CF, APPLICATION
```
Comment (DanielFernandesPTech) [inline src/main/java/com/selina/lending/internal/service/permissions/annotation/Permission.java:33]: Do we will need change keycloak or something else?

Comment (KastorDriver) [reply src/main/java/com/selina/lending/internal/service/permissions/annotation/Permission.java:33]: Yes, I already did!

Comment (KastorDriver) [reply src/main/java/com/selina/lending/internal/service/permissions/annotation/Permission.java:33]: We need to change the name of the resource and the name of the permission. I did these changes on all environments.

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **selina-finance-ms-lending-api-pr-168** [here](http://ms-lending-api-pr168.selinadev.co.uk)

File: src/main/java/com/selina/lending/internal/service/permissions/annotation/Permission.java (hunk 1 above)
Comment (DanielFernandesPTech) [reply src/main/java/com/selina/lending/internal/service/permissions/annotation/Permission.java:33]: Nice :+1:

File: src/test/java/com/selina/lending/internal/service/CreateApplicationServiceImplTest.java [hunk 2]
```diff
@@ -17,19 +17,17 @@
...
-import com.selina.lending.api.errors.custom.BadRequestException;
 import com.selina.lending.api.errors.custom.ConflictException;
 import com.selina.lending.internal.mapper.MapperBase;
 import com.selina.lending.internal.repository.MiddlewareApplicationServiceRepository;
 import com.selina.lending.internal.repository.MiddlewareRepository;
 import com.selina.lending.internal.service.application.domain.ApplicationIdentifier;
 import com.selina.lending.internal.service.application.domain.ApplicationRequest;
 import com.selina.lending.internal.service.application.domain.ApplicationResponse;
-import com.selina.lending.internal.service.application.domain.quotecc.QuickQuoteCCRequest;
+import com.selina.lending.internal.service.application.domain.quotecc.QuickQuoteCFRequest;
```
Comment (DanielFernandesPTech) [inline src/test/java/com/selina/lending/internal/service/CreateApplicationServiceImplTest.java:27]: Name package should be `quotecf` ?

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[WeiLu1](https://github.com/Selina-Finance/ms-lending-api/pull/168#issuecomment-1508334910 "LGTM")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [WeiLu1]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by selina-bot-user** [comment]
:star: PR built and available in a preview **selina-finance-ms-lending-api-pr-168** [here](http://ms-lending-api-pr168.selinadev.co.uk)

**Comment by selina-bot-user** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *[DanielFernandesPTech](https://github.com/Selina-Finance/ms-lending-api/pull/168#issuecomment-1508339074 "Approved")*, *[WeiLu1](https://github.com/Selina-Finance/ms-lending-api/pull/168#issuecomment-1508334910 "LGTM")*

The full list of commands accepted by this bot can be found [here](https://jenkins-x.io/v3/develop/reference/chatops/?repo=Selina-Finance%2Fms-lending-api).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/Selina-Finance/ms-lending-api/blob/main/OWNERS)~~ [DanielFernandesPTech,WeiLu1]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**